File: rest-api/src/main/java/com/kt/kol/api/post/model/PostDTO.java
package com.kt.kol.api.post.model;

import java.time.LocalDateTime;

public record PostDTO(
    Long id,
    String title,
    String content,
    Long userId,
    String username,
    String status,
    Integer viewCount,
    LocalDateTime createdAt,
    LocalDateTime updatedAt
) {}

File: rest-api/src/main/kotlin/com/kt/kol/api/post/model/PostDTO.kt
package com.kt.kol.api.post.model

import java.time.LocalDateTime

data class PostDTO(
    val id: Long? = null,
    val title: String? = null,
    val content: String? = null,
    val userId: Long? = null,
    val username: String? = null,
    val status: String? = null,
    val viewCount: Int? = null,
    val createdAt: LocalDateTime? = null,
    val updatedAt: LocalDateTime? = null
)